components/d2l-filter-dropdown/d2l-filter-dropdown-category.js

```diff
@@ -165,6 +165,9 @@ class FilterDropdownCategory extends LocalizeStaticMixin(TabPanelMixin(LitElemen
 			this.selected = !this.selected;
 			this.__onSelect(e);
 		});
+		this.addEventListener('d2l-tab-panel-selected', () => {
```

Comment:
I like the idea of stopping propagation of this event, but that doesn't happen currently and someone could be relying on that, so leaving for now.

Reply:
The [d2l-dropdown-tabs](https://github.com/BrightspaceUI/core/blob/master/components/dropdown/dropdown-tabs.js#L29) component appears to depend on this event.

Reply:
Which makes sense, we don't want it to rely on a specific filter event.  It's the only use-case for now, but there may be more in the future